pyproject.toml

```
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools>=77", "wheel", "setuptools_scm>=7.1.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+where = ["src/"]
+include = ["mlx.jira_juggler"]
+
+[tool.setuptools_scm]
+write_to = "src/mlx/jira_juggler/_version.py"
+
+[project]
+name = "mlx.jira_juggler"
+description = "A Python script for extracting data from Jira, and converting to TaskJuggler (tj3) output"
+authors = [
+    {name = "Jasper Craeghs", email = "jce@melexis.com"}
+]
+readme = "README.rst"
+requires-python = ">=3.9"
+license = "Apache-2.0"
+license-files = ["LICENSE"]
+dynamic = ["version"]
```

I assume entry-point we also want it to be taken from setup.py.

```suggestion
dynamic = ["version","entry-points"]
```

---

This is not a thing. I removed setup.py. https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#static-vs-dynamic-metadata